scripts/config.js

```diff
@@ -140,6 +136,21 @@ ${chalk.dim(
   `Client Secret is needed for privileged transitions with Marketplace API.
 Client secret is connected to client ID. You can find your client secret from Sharetribe Console.`
 )}
+`,
+      default: function() {
+        return rootUrlDefault ? rootUrlDefault : 'http://localhost:3000';
```

Comment:
I think you have copy-pasted the defaults wrong between 
- SHARETRIBE_SDK_CLIENT_SECRET
- REACT_APP_MARKETPLACE_NAME
- REACT_APP_MARKETPLACE_ROOT_URL